crypto-keys/src/main/java/com/palantir/crypto2/keys/serialization/SymmetricKeySerializerV3.java

```diff
@@ -52,9 +52,11 @@
 enum SymmetricKeySerializerV3 implements SymmetricKeySerializer {
     INSTANCE;
 
-    private static final String AES_CTR_NO_PADDING = "AES/CTR/NoPadding";
-    private static final int IV_SIZE = 16;
-
+    private static final String AES_GCM_NO_PADDING = "AES/GCM/NoPadding";
+    private static final int IV_SIZE = 12;
+    // 128 bit tag length as recommended by:
+    // https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-38d.pdf
```

Comment:
@cbarcenas could you take a look at this, please?

Reply:
This is standard - IVs in GCM mode are usually 92 bits wide. Inside the GCM construction, a 32-bit counter suffix is appended to the input IV to form a 128-bit-wide input to the AES block function.

Reply:
It's probably worth updating the inline comment to better clarify this

Reply:
this comment is about the authentication tag, not the IV. I can add your comment to the IV_SIZE constant if you'd like.